src/components/User/User.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './User.scss';
+import PropTypes from 'prop-types';
+
+export const User = ({ user }) => (
+  <h2 className="todo-list__user-name">{user.name}</h2>
+);
+
+User.propTypes = {
+  user: PropTypes.objectOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+    }),
+  ),
```

Comment:
объекты описываются через shape(), objectOf не нужен
```suggestion
  user: PropTypes.shape({
      name: PropTypes.string.isRequired,
    }),
```